src/views/apply/dashboard.html

```diff
@@ -0,0 +1,104 @@
+{% extends 'default.html' %}
+{% set title = 'Apply for Hack Cambridge' %}
+{% set is_dashboard = true %}
+{% block pageHeading %}Dashboard{% endblock %}
+{% block pageContent %}
+<header class="content-wrapper-center">
+  <h3>Hello, {{ user.first_name }}!</h3>
+</header>
+
+  {# The main status banner #}
+  <div class="content-wrapper-center">
+    {% if applicationStatus == 'incomplete' %}
```

Comment:
I now see what you were talking about with making a yaml file to encode this stuff.

I think that's a good way forward with this (or even just objects in js). Templates aren't a good data structure to encode this information.